scripts/pre_push_hook_test.py

```diff
@@ -211,7 +239,9 @@ def test_get_remote_name_with_multiple_remotes_set(self):
         process_for_origin_url = subprocess.Popen(
             [b'echo', b'url.oppia/oppia.git'], stdout=subprocess.PIPE,
             stderr=subprocess.PIPE)
-        def mock_popen(cmd_tokens, stdout, stderr):  # pylint: disable=unused-argument
+        def mock_popen(
+            cmd_tokens: List[bytes], stdout: int, stderr: int  # pylint: disable=unused-argument
```

Comment:
Maybe use `unused_` and drop the pylint pragma.

Reply:
We cannot do this because we providing stdout and stderr as a keyword argument
https://github.com/oppia/oppia/blob/e03494900a56d5e0d1470137c8b9b75d53445114/scripts/pre_push_hook.py#L126
If we use `unused_` prefix then a mismatch will occur and the backend test will fail.